packages/toolkit/package.json

```
@@ -40,7 +40,7 @@
     "eslint-webpack-plugin": "^3.1.1",
     "html-webpack-plugin": "^5.5.0",
     "ignore-emit-webpack-plugin": "2.0.6",
-    "image-minimizer-webpack-plugin": "^3.2.3",
+    "image-minimizer-webpack-plugin": "3.2.1",
```

@darylldoyle sorry we didn't get back to this but I've been thinking more about this. I think it makes sense to leave the the `^` so most users can benefit from updates and if someone runs into the issue they can just _pin_ the dependency in their package.json without causing any NPM errors. 

This means we would need to add some documentation around this. Are you keen to do that or would you rather want to leave that out of the PR?

```suggestion
    "image-minimizer-webpack-plugin": "^3.2.1",
```